lib/livebook_web/live/session_live/app_docker_component.ex

```diff
@@ -6,17 +6,23 @@ defmodule LivebookWeb.SessionLive.AppDockerComponent do
   alias Livebook.Hubs
   alias Livebook.FileSystem
   alias LivebookWeb.AppHelpers
+  alias Livebook.Teams
 
   @impl true
   def update(assigns, socket) do
     socket = assign(socket, assigns)
+    hub_type = Hubs.Provider.type(assigns.hub)
+    deployment_groups = if hub_type == "team", do: Teams.get_deployment_groups(assigns.hub)
```

Comment:
Same comment here. In theory other parts of the app should not know about Teams. Maybe we need a `Provider.deployment_groups` which returns a map from deployment group "Name" to "IDs" (or vice-versa)? If there is no deployment group (personal hub), we return `nil`?